src/main/java/frc/robot/utils/VisionDataProcessor.java

```diff
@@ -42,7 +42,7 @@ public void addInput(String readings) {
     }
 
     private boolean completeData(String data) {
```

Comment:
the cod3 around all this parsing is really nasty. 
if it works, lets leave it, but there must be a better way.
i think we can look at re-doing it after palmetto-- and also at that time work in getting frames from the camera. I'm concluding i'll need to be able to send commands to the camera so i know what i'm getting, and that will complicate things